src/models/Orders.js

```diff
@@ -0,0 +1,105 @@
+const mongoose = require('mongoose');
+
+// mongoose schema for the individual order items
+const orderItemSchema = new mongoose.Schema({
+    // The album field references the Album model and is required
+  album: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Album', // Referencing the Album model
+    required: true,
+  },
+  // The quantity field specifies the number of items in the order
+  quantity: {
+    type: Number,
+    required: true, // Quantity is required
+    min: 1, // Minimum quantity allowed is 1
+  },
+});
+
+// Create the OrderItem model using the defined schema
+const OrderItem = mongoose.model('OrderItem', orderItemSchema);
+
+// Schema for the main Order model
+const orderSchema = new mongoose.Schema({
```

Comment:
The order schema is missing the `orderItems` as an embedded part of the document. That way we can easily know what was ordered as part of a given Order:

```javascript
const orderSchema = new mongoose.Schema({
	... all the existing stuf...,
	orderItems: [OrderItemSchema],
})
```

Reply:
<img width="506" alt="orderItems" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/95896159/80174908-88a3-4f39-8c85-6397a0bbb756">